cmd/process-agent/collector_api_test.go

```diff
@@ -338,9 +351,12 @@ func TestRTProcMessageNotRetried(t *testing.T) {
 func TestSendPodMessageSendManifestPayload(t *testing.T) {
 	clusterID, cfg, check := getPodCheckMessage(t)
 
-	init := func(c *Collector) {
+	init := func(c *Collector, s *submitter) {
 		c.orchestrator.OrchestrationCollectionEnabled = true
 		c.orchestrator.IsManifestCollectionEnabled = true
+
+		s.orchestrator.OrchestrationCollectionEnabled = true
+		s.orchestrator.IsManifestCollectionEnabled = true
```

Comment:
We already use config directly in some other tests, I wonder if here we could do that too? This would be:
```
config.Set("orchestrator_explorer.enabled", true)
config.Set("orchestrator_explorer.manifest_collection.enabled", true)
```
Pro of this - it would remove the need for having these sneaky `init` funcs.
Con - it would be requiring we use a mock config, but this feels OKish.
Thoughts @just-chillin?

Reply:
TBH I don't really consider requiring the use of a mock config a con at all here. I went ahead and removed the init function in favor of using the global config.